react.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     'react/destructuring-assignment': [OFF, 'always'],
     'react-hooks/rules-of-hooks': ERROR,
     'react-hooks/exhaustive-deps': WARNING,
+    'no-unused-vars': ERROR,
```

Comment:
You should include this in `node.js` config since it's the base for all the others.